content/en/docs/ibm/pipelines.md

```diff
@@ -4,11 +4,51 @@ description = "Instructions for using Kubeflow Pipelines on IBM Cloud Kubernetes
 weight = 50
 +++
 
-## Authenticating Kubeflow Pipelines with the SDK
+By default, Kubeflow pipelines on IBM Cloud are running with the Tekton backend. Below are the instructions on how to use the Kubeflow Pipelines with the Tekton backend [(kfp-tekton)](https://github.com/kubeflow/kfp-tekton).
```

Comment:
Can you just double check that it's KF pipelines and not KFP with a capital "P" here @Tomcli in the first sentence. Or maybe it is, based on the second sentence 😄 
```suggestion
By default, Kubeflow Pipelines on IBM Cloud are running with the Tekton backend. In this guide you'll learn how to use the Kubeflow Pipelines with the Tekton backend [(kfp-tekton)](https://github.com/kubeflow/kfp-tekton).
```

(this is probably more "why the user should pay attention" with more "you" here)